app/views/SidebarView/index.js

```diff
@@ -142,6 +154,17 @@ class Sidebar extends Component {
 		});
 	}
 
+	setStatusText = async(statusText) => {
+		const result = await RocketChat.saveUserProfile({ statusText }, {});
+		if (result.success) {
+			EventEmitter.emit(LISTENER, { message: I18n.t('Status_saved_successfully') });
```

Comment:
Is there a reason why these are not consistent? "Status_save..." vs "error-could".

Reply:
Actually, scrap that. It looks like it is consistent with what is in the i18n file. Nevermind.